src/main/java/com/example/pokebattlez/battle/service/BattleService.java

```diff
@@ -1,14 +1,23 @@
 package com.example.pokebattlez.battle.service;
 
 import com.example.pokebattlez.battle.controller.Battle;
+import com.example.pokebattlez.battle.model.Pokemon;
+import com.example.pokebattlez.battle.model.packet.BattleState;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Service;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 @Service
 public class BattleService {
```

Comment:
What is BattleService meant to do? How it's logic is meant to be different from the Battle object itself?

Reply:
BattleService is meant to manage multiple instances of Battle.